modules/statics/src/base.ts

```diff
@@ -1376,6 +1380,8 @@ export enum BaseUnit {
   KAVA = 'ukava',
   CORE = 'ucore',
   TESTCORE = 'utestcore', // Coreum testnet uses different name for native coin
+  BERA = 'abera',
+  ISLM = 'aISLM',
```

Comment:
why this is in capital case? all others are in small case

Reply:
That's just the format expected by this particular chain, so we'll need it that way